### ISSUE TITLE
[A11y] Jaws is not announcing column header name while navigating to table cells 

### ISSUE THREAD

**Issue by theot12** [NuGet/NuGetGallery#9471]
URL: [1745273](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1745273)

Note: 
Issue is not observed with NVDA.
Similar issue observed in below scenario
Hit the URL: https://www.nuget.org/
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Search for a key term (say “json”) from search box.
Click on "Newtonsoft.json" control and hit enter.
Navigate to "Used By" control and hit enter.
Observe whether the screen reader is announcing the column headers for the column header names or not.
Expected Result:
While navigating in a table using table shortcut keys(Ctrl+Alt+arrow keys), screen readers should read corresponding column header name when we are moving from column to other column

Example: Pattern Column1 * , Result Column 2 Select any package that has an ID beginning with Alpha

Environment Details:
App name: NuGet Service
#URL: [https://www.nuget.org](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.nuget.org%2f&data=05%7c01%7cv-nitipi%40microsoft.com%7c9a84307e7aa744a4b24208dafa5ca2fe%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638097573052590837%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=C8Eab3i6ixAmqsOjG4AxNT36z9WZQiXPbjDDkMcS5ME%3D&reserved=0)
OS and Browser Details:
Edition Windows 11 Enterprise
OS build 22621.1105.
Chrome version:  109.0.5414.120 (Official Build) (64-bit)
Screen reader: JAWS, NVDA

Repro Steps:
Hit the URL: https://www.nuget.org/
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Select “API Keys” from the Profile Dropdown and hit enter.
Navigate to "Create" button and hit enter and Unselect Push Checkbox.
Observe whether the screen reader is announcing the column headers for the column header names or not.

User Impact:
Screen reader users will not be able to get the information of the table clearly.

Recommendations:
Refer [Repository of bug fixes code snippets](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faka.ms%2faccsnippet&data=05%7c01%7cv-samhithaks%40microsoft.com%7c535b4a834ef843acdfb508da53769f91%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637914066217264155%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=xoIfLB9Iq6HVIPyUc42rS%2BlVQCt9m%2BaUokp32Kaz1nE%3D&reserved=0)

**Comment by theot12** [comment]
Cannot reproduce. Sent back to tester.

**Comment by theot12** [comment]
AzDo issue closed.
